studio/data/invoices/project-invoices-query.ts

```diff
@@ -0,0 +1,51 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { get } from 'data/fetchers'
+import { ResponseError } from 'types'
+import { invoicesKeys } from './keys'
+
+export type ProjectInvoicesVariables = {
+  projectRef?: string
+  offset: number
+  limit: number
```

Comment:
Seems strange to me that these are required – can't we provide defaults? e.g. `offset = 0` and `limit = 50`?

Reply:
ahh yeap makes sense